src/config.ts

```diff
@@ -80,8 +80,20 @@ export async function getConfigFile(config: ScaffoldCmdConfig, tmpPath: string):
 }
 
 /** @internal */
-export async function parseConfigFile(config: ScaffoldCmdConfig, tmpPath: string): Promise<ScaffoldConfig> {
-  let output: ScaffoldConfig = { ...config, beforeWrite: undefined }
+export async function parseConfigFile(config: ScaffoldCmdConfig): Promise<ScaffoldConfig> {
+  let output: ScaffoldConfig = {
+    name: config.name,
+    templates: [],
```

Comment:
It appears this change may have broken the templates.  Previously, if a user defined a template, it would pass through with the parsed config, but afterward the templates is being reset to an empty array, so its not matching on any files.

Reply:
Just so I understand fully what you mean, when you say "user defined", do you mean via the CLI args, trying to add data to the template config?

Or the config itself loses the templates?

Reply:
Created a new issue to track this regression

https://github.com/chenasraf/simple-scaffold/issues/110

Let's continue communication there so it's easier to track